### DISCUSSION TITLE
deploy error nuxt3

### DISCUSSION THREAD

**Discussion by bang98ptit** [nuxt/nuxt#17372] [Questions]
when i run: yarn start and enter link on browser then this page display: 500 Cannot read properties of null (reading 'setupContext')
How can i fix?

Image:
https://user-images.githubusercontent.com/51089062/176122513-eb385f59-db63-45e5-9bdf-61445da5814d.png

**Comment by zihaoyy** [comment]
> 

This error means that the value of the setupContext you defined is null. It is recommended to preset the judgment in the production environment. If it is empty, set the default value, so that no error will be reported.

**Comment by bang98ptit** [comment]
So what file i need to  edit, i can not to find this? Thanks

Vào Th 4, 29 thg 6, 2022 lúc 06:29 Zihao ***@***.***> đã viết:

**Reply by zihaoyy** [reply]
You can fix it by searching for the keyword 'setupContext' globally. If the value of the variable is an array, you can use the ternary operator to set the default null value to prevent an error when the value is null

**Comment by bang98ptit** [comment]
can you give me your social network account? i have some question which
need you answering. Thanks

Vào Th 4, 29 thg 6, 2022 vào lúc 07:19 Zihao ***@***.***>
đã viết:
